src/olmo_core/data/numpy_dataset.py

```diff
@@ -2307,6 +2308,10 @@ class NumpyDatasetConfig(Config, ABC):
     """
     The tokenizer config.
     """
+    sequence_length: Optional[int] = None
+    """
+    The sequence length for a :class:`NumpyFSLDataset`.
+    """
```

Comment:
Can we remove this? Not all subclasses use it (eg `NumpyVSLDatasetConfig`). All the relevant subclasses should have it defined.